README.md

```diff
@@ -1 +1,82 @@
-# -Admissions-Trial
```

Comment:
Repetition of the project name.

Reply:
Please make sure that your comments are not vague, instead, they should be descriptive and pointing to a solution.

Reply:
- Great Job stating reasons WHY  a change needs to be implemented, but I think it will be better if your phrase your comments to sound a bit more polite. From your comments above, it sounds like you are giving an order. 

Just as @wale-prog suggested earlier, I think you can re phrase to something like this 👉🏽 _"I think this topic was repeated twice, I'd like to kindly suggest that it is important to always cross-check your work to avoid repetitions and typographic errors"_

What do you think about how I have re written your comment?